contracts/upgradeable_contracts/BasicTokenBridge.sol

```diff
@@ -90,12 +90,12 @@ contract BasicTokenBridge is EternalStorage, Ownable {
     }
 
     function setMaxPerTx(uint256 _maxPerTx) external onlyOwner {
-        require(_maxPerTx < dailyLimit());
+        require(_maxPerTx == 0 || (_maxPerTx > minPerTx() && _maxPerTx < dailyLimit()));
         uintStorage[MAX_PER_TX] = _maxPerTx;
     }
 
     function setMinPerTx(uint256 _minPerTx) external onlyOwner {
-        require(_minPerTx < dailyLimit() && _minPerTx < maxPerTx());
+        require(_minPerTx > 0 && _minPerTx < dailyLimit() && _minPerTx < maxPerTx());
```

Comment:
Given that the limits are consistently set by initialization method, `setMaxPerTx` and `setDailyLimit`, is it ensured that `maxPerTx() < dailyLimit()`. Does it makes sense to only check that `_minPerTx < maxPerTx()`?
```suggestion
        require(_minPerTx > 0 && _minPerTx < maxPerTx());
```

Reply:
If we make it so, then I guess that we will be able to change `minPerTx` limit when the bridge is stopped by `setDailyLimit(0)`, while at the same time, we won't be able to change it when the bridge is stopped by `setMaxPerTx(0)`.
Should it behave like this?

Reply:
I see. I think we should keep the implementation as it is for consistency, so no other limits can be updated when the bridge is stopped. Thanks